design-toolkit/components/src/tokens/generated/themes.css

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/*
+  This file is generated by the generate-tokens.mjs script.
+  Do not edit this file manually.
+ */
+
+@import './tokens.css';
+
+@theme {
+  /* Remove the default Tailwind styles. */
+  --color-* : initial;
+  --font-* : initial;
+  --radius-* : initial;
+  --shadow-* : initial;
+  --spacing-* : initial;
+  --text-* : initial;
+}
+
+@theme {
+  /** Colors (semantic) **/
+  --color-surface-default: var(--colors-neutral-10, #0b0b0b);
```

Comment:
programmatically mapped via `semantics.json` mapping; one `@theme {} ` block per each top level key in `tokens.json`